### PR TITLE
pink-drivers: add query interfaces

### PR DESCRIPTION
**Pull request by shelvenzhou**

### CODE REVIEW

File: crates/pink-drivers/system/lib.rs [hunk 1]
```diff
@@ -50,6 +50,11 @@ mod system {
             Err(Error::BadOrigin)
         }
 
+        #[ink(message)]
+        pub fn is_admin(&self, contract_id: AccountId) -> Result<bool> {
```
Comment (kvinwang) [inline crates/pink-drivers/system/lib.rs:54]: Should we move it into the `trait System`?

Comment (shelvenzhou) [reply crates/pink-drivers/system/lib.rs:54]: sure, let me do that

**Comment by kvinwang** [comment]
I've turned the Clippy check into a required condition to merge.
Please rebase onto the master if the Clippy check fails.

File: crates/pink-drivers/system/lib.rs (hunk 1 above)
Comment (kvinwang) [reply crates/pink-drivers/system/lib.rs:54]: Let me do it if you haven't time. OK?

**Review by kvinwang — APPROVED** [review]

File: crates/pink-drivers/system/lib.rs (hunk 1 above)
Comment (shelvenzhou) [reply crates/pink-drivers/system/lib.rs]: Yes, pls. Thx.

**Pull request merged**